frodo.js

```diff
@@ -1,17 +1,18 @@
 /**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.